src/domain/Buttons/FontAwesomeIconButton/style.ts

```diff
@@ -14,12 +17,51 @@ interface IStyledIconButton {
   size: FontAwesomeIconButtonSize
 }
 
+interface ITooltipPopoverWrapper {
+  size: FontAwesomeIconButtonSize
+}
+
+interface IStyledStatusDot {
+  variant: StatusDotVariants
+}
+
+console.log(statusDotVariantsOptions)
+
+const StyledStatusDot = styled.span`
+  border-radius: 50%;
+  position: absolute;
+  bottom: ${Variables.Spacing.s2XSmall}px;
+  padding: ${Variables.Spacing.s2XSmall}px;
+  right: ${Variables.Spacing.s2XSmall}px;
+  border: 2px solid ${Variables.Color.n100};
+  ${(props: IStyledStatusDot) => css`
+    background: ${statusDotVariantsOptions[props.variant]};
+  `};
+`
+
 const iconButtonHoverStyle = (props: IStyledIconButton) => css`
   color: ${variantOptions[props.variant].hoverIconColor};
   background: ${variantOptions[props.variant].hoverBackground};
   transition: 0.2s ease-out;
 `
 
+const TooltipPopoverWrapper = styled.span`
+  position: relative;
+  display: inline-block;
+  width: ${Variables.Spacing.sXLarge}px;
+  height: ${Variables.Spacing.sXLarge}px;
+
+  ${(props: ITooltipPopoverWrapper) => props.size === FontAwesomeIconButton.Size.Medium && css`
+    width: ${Variables.Spacing.s2XLarge}px;
+    height: ${Variables.Spacing.s2XLarge}px;
+  `}
+
+  ${(props: ITooltipPopoverWrapper) => props.size === FontAwesomeIconButton.Size.Large && css`
+    width: ${Variables.Spacing.s3XLarge}px;
+    height: ${Variables.Spacing.s3XLarge}px;
```

Comment:
I still really don't like the spacing variables being used for things that aren't spacing but since the rest of the file is already like this so just keep it how it is 😧